docs/src/shared/utilities.ts

```diff
@@ -0,0 +1,25 @@
+function fallbackCopyTextToClipboard(text: string): void {
```

Comment:
If you are getting this code snippet from Stackoverflow, you are required to attribute the original author by at least linking back to the source answer, as all their answers are licensed under CC-BY-SA 4.0. We don't want to be plagiarizing.

Reply:
Not sure why this fallback implementation is necessary - we don't support Internet Explorer & the browser's `navigator.clipboard.writeText` method should work on all major browsers: https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText